Budget_Project/Budget_Project/settings.py

```diff
@@ -37,6 +37,7 @@
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
+    'rest_framework',
```

Comment:
tu jeszcze powinna być chyba wspomniana nasz apka?
```suggestion
    'rest_framework',
    'Budget_Application',
```